test/PotRateSource.t.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { PotRateSource } from "../src/PotRateSource.sol";
+
+contract PotMock {
+
+    uint256 public dsr;
+
+    constructor(uint256 _dsr) {
+        dsr = _dsr;
+    }
+
+    function setDSR(uint256 _dsr) external {
+        dsr = _dsr;
+    }
+
+}
+
+contract PotRateSourceTest is Test {
+
+    uint256 constant FIVE_PCT_APY_DSR = 1.000000001547125957863212448e27;
```

Comment:
Lets add a quick comment like we used to in the mainnet spells:

```
bc -l <<< 'scale=27; e( l(1.05)/(60 * 60 * 24 * 365) )'
```

Reply:
done